.asf.yaml

```diff
@@ -60,14 +60,7 @@ github:
         strict: false
         # contexts are the names of checks that must pass
         contexts:
-          - Docker/build
-          - Frontend/build
-          - PR Lint/check
-          - E2E/Cypress
-          - Python Misc/lint
-          - Python MySQL/test-mysql
-          - Python MySQL/test-postgres
-          - Python MySQL/test-sqlite
+          - PR Lint / check
```

Comment:
based on how @ktmud's example, it looks like this is the proper format, and removed the others for now if we wanted to just test one.

Reply:
Sorry, I think you need to remove the "PR Lint / " prefix, too. The `contexts` config will only match the job name, not the workflow name.

See  here the checks for: `Python MySQL / test-mysql (3.7)`: https://github.com/NUDA5020/superset/pull/1

This is what I sent to the API:

```json
{
	"required_status_checks": {
		"strict": true,
		"contexts": [
			"build",
			"lint",
			"PR Lint / check",
			"check",
			"Python MySQL/test-mysql",
		  "Python MySQL/test-mysql (3.7)",
			"abc/def"
		]
	}
}
```